observability/opentelemetry/src/main/java/org/wildfly/extension/opentelemetry/OpenTelemetrySubsystemRegistrar.java

```diff
@@ -60,13 +70,14 @@ class OpenTelemetrySubsystemDefinition extends PersistentResourceDefinition {
     };
 
     static final RuntimeCapability<Void> OPENTELEMETRY_CAPABILITY =
-            RuntimeCapability.Builder.of(OPENTELEMETRY_CAPABILITY_NAME)
+            RuntimeCapability.Builder.of(OPENTELEMETRY_CAPABILITY_NAME, Void.class)
```

Comment:
This implies that the "org.wildfly.extension.opentelemetry" capability provides a "Void" via a service.  However, no such service installed by your ResourceOperationRuntimeHandler.

Reply:
I added that... for some reason. Did that second param used to be required? Removing it, though, doesn't seem to break anything. That said, I've removed the capability altogether, and am testing that.

Reply:
Turns out that capability is used by the MP Telemetry subsystem to make sure the OpenTelemetry system is installed.

Reply:
OK - in that case, it should not declare a service type.

Reply:
Fixed.